### PR TITLE
Release 4.8.0 (#145)

### PR DESCRIPTION
**Pull request by blackbaud-svossen**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-release-4.8.0](https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-release-4.8.0)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 4.8.0 (2020-11-05)
+
+- Added country field testing fixture. [#145](https://github.com/blackbaud/skyux-lookup/pull/145)
+- Added input box to lookup demos. [#144](https://github.com/blackbaud/skyux-lookup/pull/144)
```
Comment (Blackbaud-AlexKingman) [inline CHANGELOG.md:4]: You don't need to include any information on the demo/docs - that can stay internal.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# 4.8.0 (2020-11-05)
+
+- Added country field testing fixture. [#145](https://github.com/blackbaud/skyux-lookup/pull/145)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:3]:
```suggestion
- Added a country field testing fixture. [#145](https://github.com/blackbaud/skyux-lookup/pull/145)
```

**Comment by blackbaud-svossen** [comment]
@Blackbaud-AlexKingman @blackbaud-johnly Comments should be addressed, thanks.

**Review by blackbaud-johnly — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-release-4.8.0](https://developer.blackbaud.com/skyux-lookup?_sp.spa=pr-release-4.8.0)

**Pull request merged**